roles/branch-validation/tasks/main.yml

```diff
@@ -113,6 +125,7 @@
   delay: 1
   retries: 5
   with_items: instances.results
+  when: not groups['compute'] | intersect(groups['controller']) | length | bool
```

Comment:
Why is this conditional here? The only things inside of `instances.results` would be the instances we spun up, one for every computer in the `instance_group` group (which, we should go back to making that hard set to `compute` since there isn't a way in Ursula for this group to be anything different).